packages/anac-certified-attributes-importer/test/download-csv.test.ts

```diff
@@ -0,0 +1,128 @@
+import sftp from "ssh2-sftp-client";
+import { describe, expect, it } from "vitest";
+import { SftpClient } from "../src/service/sftpService.js";
+import { sftpConfigTest } from "./helpers.js";
+
+describe("CSV getFileName", () => {
+  const fileInfoMock: sftp.FileInfo = {
+    type: "-",
+    name: "",
+    size: 0,
+    modifyTime: 0,
+    accessTime: 0,
+    rights: {
+      user: "user",
+      group: "group",
+      other: "other",
+    },
+    owner: 0,
+    group: 0,
+  };
+
+  it("should retrieve last file (alphabetical order)", async () => {
+    const sftpClient = new SftpClient({
+      ...sftpConfigTest,
+      forceFileName: undefined,
+    });
+
+    const fileList: sftp.FileInfo[] = [
+      {
+        ...fileInfoMock,
+        name: `${sftpConfigTest.fileNamePrefix}-22221100.csv`,
+      },
```

Comment:
following the previous comment you can simply use
```suggestion
      getMockFileInfo(`${sftpConfigTest.fileNamePrefix}-22221100.csv`),      
```